lib/3scale/api/client.rb

```diff
@@ -623,6 +623,46 @@ def delete_service_feature(service_id, id)
         true
       end
 
+      # @api public
+      # @return [Array<Hash>]
+      def list_policy_registry
+        response = http_client.get('/admin/api/registry/policies')
+        extract(collection: 'policies', entity: 'policy', from: response)
+      end
+
+      # @api public
+      # @param [Hash] attributes Policy Registry Attributes
+      # @return [Hash]
+      def create_policy_registry(attributes)
+        response = http_client.post('/admin/api/registry/policies', body: attributes)
+        extract(entity: 'policy', from: response)
+      end
+
+      # @api public
+      # @return [Hash]
+      # @param [Fixnum] id Policy Registry Id
+      def show_policy_registry(id)
+        response = http_client.get("/admin/api/registry/policies/#{id}")
+        extract(entity: 'policy', from: response)
+      end
+
+      # @api public
+      # @return [Hash]
+      # @param [Fixnum] id Policy Registry Id
+      # @param [Hash] attributes Policy Registry Attributes
+      def update_policy_registry(id, attributes)
+        response = http_client.put("/admin/api/registry/policies/#{id}", body: attributes)
+        extract(entity: 'policy', from: response)
+      end
+
+      # @api public
+      # @param [Fixnum] id Policy Registry Id
+      # @param [Hash] attributes Policy Registry Attributes
```

Comment:
This line was copy-pasted? I think it does not apply here

Reply:
fixed